portfolio/src/main/webapp/index.html

```diff
@@ -23,7 +23,9 @@ <h1>a page about me</h1>
                 <br>
                 <a href="https://github.com/biancamacias/" target="_blank">Github</a>
                 <br>
-                <p id="quote-container" onclick="promptRandomQuote()">daily quote generator!</p>
+                <div id="quoteDisplayer">
+                </div>
+                <button id="quote-button" onclick="popupMessage()">daily quote generator!</button>
```

Comment:
I think the statement inside the onclick event should be a complete statement so it should end with a semi-colon. I'm not 100% sure though.
```suggestion
                <button id="quote-button" onclick="popupMessage();">daily quote generator!</button>
```